modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyAssertions.ts

```diff
@@ -16,8 +16,8 @@ const assertPath = (label: string, root: SugarElement, expPath: number[], expOff
   Assertions.assertEq(() => 'Offset mismatch for ' + label + ' in :\n' + Html.getOuter(expected), expOffset, actOffset);
 };
 
-const assertContent = (editor: Editor, expected: string): void => {
-  const content = editor.getContent();
+const assertContent = (editor: Editor, expected: string, args?: object): void => {
```

Comment:
This should be the `GetContentArgs` type not `object`. Also worth noting using `object` is generally discouraged and instead `Record<string, any>` is preferred.